### PR TITLE
WS2-1672 - local development helper: Allow select repo files to be easily blocked locally.

### PR DESCRIPTION
**Pull request by bjdevil21**
### Description

Allow developers to block certain repo files locally by adding a .gitignore_local -> .git/info/exclude symbolic link.

### Links

- [JIRA ticket](https://asudev.jira.com/browse/WS2-1672)

### CODE REVIEW

**Comment by bjdevil21** [comment]
Multidev cap of 10 has been reached, so there’s no testing ENV on Pantheon to check.

Despite this, there's nothing really to check there anyways since this is all about local dev workflows.

**Comment by bjdevil21** [comment]
Re-uploaded the .gitignore file after fixing line endings.

**Comment by mlsamuelson** [comment]
[Visit Site](https://pr-35-webspark-ci.pantheonsite.io/)

Created multidev environment [pr-35](https://dashboard.pantheon.io/sites/aec2f75f-35eb-41f8-abe7-95b292415259#pr-35) for webspark-ci.

File: .gitignore [hunk 1]
```diff
@@ -1,8 +1,8 @@
-# :::::::::::::::::::::: cut ::::::::::::::::::::::
-
-# Put ignore patterns for build artifacts above the cut line
-# so they are ignored in the source repo and committed
-# to Pantheon.
+# :::::::::::::::::::::: cut ::::::::::::::::::::::
```
Comment (bebaps) [inline .gitignore:1]: Just curious, what is this for?

Comment (bjdevil21) [reply .gitignore:1]: I'd just left it in from it existing from other code. It's not there for me.

**Comment by bjdevil21** [comment]
Added to WS2-1519 to avoid merge conflicts. Closing.

**Pull request closed without merging**